openapi/toy-dns.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
What's the difference between this toy-dns.json and the one at the top level?

Reply:
The difference is me forgetting to `mv toy-dns.json openapi/` after generating an update version :/

Reply:
Fixed in d8a9c22